<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumePlayer</class>
 <widget class="QDockWidget" name="VolumePlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>158</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFeatureMask</set>
  </property>
  <property name="windowTitle">
   <string>Volume Player</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="VolumePlayerWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="VolumePlayerLayout" stretch="0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QSlider" name="volumeSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBelow</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="PlayerLowerLayout">
           <item>
            <layout class="QHBoxLayout" name="ButtonsLayout">
             <item>
              <widget class="QToolButton" name="tbPlay">
               <property name="toolTip">
                <string>Play</string>
               </property>
               <property name="text">
                <string>Play</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbPause">
               <property name="toolTip">
                <string>Pause</string>
               </property>
               <property name="text">
                <string>Pause</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbStop">
               <property name="toolTip">
                <string>Stop</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbPrev">
               <property name="toolTip">
                <string>Previous</string>
               </property>
               <property name="text">
                <string>Previous</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="tbNext">
               <property name="text">
                <string>Next</string>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QCheckBox" name="blending">
             <property name="text">
              <string>Blending</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="SpeedLayout">
             <item>
              <widget class="QSlider" name="speedSlider">
               <property name="maximum">
                <number>999</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="tickPosition">
                <enum>QSlider::TicksBelow</enum>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Speed:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QDoubleSpinBox" name="sbSpeed">
                 <property name="decimals">
                  <number>2</number>
                 </property>
                 <property name="maximum">
                  <double>50.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                 <property name="value">
                  <double>1.000000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QToolButton" name="tbApplyForAll">
             <property name="text">
              <string>Apply transfer function
for all</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="dataTable">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
